hydra-node/src/Hydra/Node.hs

```diff
@@ -203,11 +204,13 @@ processEffect HydraNode{hn, oc = Chain{postTx}, server, eq, env = Environment{pa
 -- alternative implementation
 data EventQueue m e = EventQueue
```

Comment:
Should: move this and all related functions to a dedicated module `Hydra.Node.EventQueue` as you have a corresponding `Spec` module now.